src/nxdoc/nuxeo-server/installation/docker-image/nuxeo-kubernetes-deployment.md

```diff
@@ -0,0 +1,183 @@
+
+### Deployment and StatefulSets
+
+There are different ways to deploy an application / pods on Kubernetes using different Kubernetes resources. Below are two different resources that Kubernetes provides for deploying pods.
+
+#### Deployment
+
+A [Kubernetes Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) is used to tell Kubernetes how to create or modify instances of the pods that hold a containerized application. Deployments can scale the number of replica pods, enable rollout of updated code in a controlled manner, or roll back to an earlier deployment version if necessary. 
+
+#### StatefulSets
+
+A [Kubernetes StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) Manages the deployment and scaling of a set of Pods, and provides guarantees about the ordering and uniqueness of these Pods. StatefulSets are valuable for applications that require one or more of the following.
+
+- Stable, unique network identifiers.
+- Stable, persistent storage.
+- Ordered, graceful deployment and scaling.
+- Ordered, automated rolling updates
+
+{{#> callout type='info' heading="recommandation"}}
+You can **either use Deployments or StatefulSets** to deploy your Nuxeo application. 
+{{/callout}}
+
+### Service
+
+A [Kubernetes service](https://kubernetes.io/docs/concepts/services-networking/service/) is a logical abstraction for a deployed group of pods in a cluster (which all perform the same function). Since pods are ephemeral, a service enables a group of pods, which provide specific functions (web services, image processing, etc.) to be assigned a name and unique IP address (clusterIP).
+
+### ConfigMap and Kubernetes Secrets
+
+[Kubernetes ConfigMaps](https://kubernetes.io/docs/concepts/configuration/configmap/) are Kubernetes objects that allow you to separate configuration data/files from image content to keep containerized applications portable. ConfigMaps bind configuration files, command-line arguments, surroundings variables, port numbers, and alternative configuration artifacts to your Pods containers and system parts at run-time. ConfigMaps are helpful for storing and sharing non-sensitive, unencrypted configuration data. 
+
+If you need to store and manage sensitive information, such as passwords, OAuth tokens, and ssh keys, then you'll have to use [Kubernetes Secrets]() Storing confidential information in a Secret is safer and more flexible than putting it verbatim in a Pod definition or in a container image. 
+
+ConfigMaps and Secrets can be produced from files and with yaml declaration.
+
+{{#> callout type='info' heading="recommandation"}}
+<br/>
+- **Do not store sensitive data**, such as passwords, authentication tokens and SSH keys, **in plaintext on a container**.
```

Comment:
```suggestion
- **Do not store sensitive data**, such as passwords, authentication tokens or SSH keys **in plaintext**.
```

In the end, password endup in plaintext in the container, but they should not be stored as configuration map